doc-Monitoring_Alerts_and_Reporting/topics/Operational_Alerts.adoc

```diff
@@ -51,7 +51,7 @@
 					
 | 
 						
-							Alert is raised when one EVM server detects that another EVM Server has not responded in (2 minutes).
+							Alert is raised when one server detects that another server has not responded in (2 minutes).
```

Comment:
I think the brackets around (2 minutes) are probably best to be removed here. What do you think?